app/assets/src/components/common/DetailsSidebar/GeneDetailsMode/gene_details_mode.scss

```diff
@@ -0,0 +1,74 @@
+@import "~styles/typography";
+@import "~styles/themes/colors";
+
+.content {
+  padding: 30px 20px;
+  position: relative;
+}
+
+.title,
+.loadingMsg {
+  font-size: $font-size-h4;
+  font-weight: $font-weight-semibold;
+  margin: 0 10px;
+  margin-right: 60px;
+  word-break: break-all;
+  line-height: 1.4em;
+}
+
+.geneContents {
+  padding: 0 10px;
+
+  .subtitle {
+    padding-top: 25px;
+    padding-bottom: 10px;
+    font-size: 14px;
+    font-weight: 700;
+  }
+
+  .text {
+    line-height: 24px;
+  }
+
+  .linksSection {
+    &:after {
+      content: "";
+      display: table;
+      clear: both;
+    }
+  }
+
+  .linksList {
+    min-width: 200px;
+    float: left;
+    list-style: none;
+  }
+
+  .link {
+    color: $primary-light;
+    line-height: 30px;
+    cursor: pointer;
+  }
+
+  .expandLink {
+    color: $primary-light;
+    margin-top: 10px;
+    cursor: pointer;
+  }
+}
+
+.cardLicense {
+  padding-top: 10px;
+  color: $medium-grey;
+  font-size: $font-size-small;
```

Comment:
We're pushing to use the mixins from `styles/themes/_typography` now. You can use them like `@include font-body-xs;`

`font-size-small` and all those variables are deprecated.